databroker/_core.py

```diff
@@ -2087,7 +2094,7 @@ class Broker(BrokerES):
     name : str, optional
         The name of the broker
     """
-    def __init__(self, mds, reg=None, plugins=None, filters=None,
+    def __init__(self, mds, reg=None, plugins=None, filters=None, event_sources = None,
```

Comment:
put `event_sources` last so that we do not beak API of people using many positional arguements.

Reply:
This should be a list of EventSources or None